pkg/transaction/metric/metricscollector.go

```diff
@@ -695,8 +695,10 @@ func (c *collector) generateUsageEvent(orgGUID string) {
 			},
 		},
 		Meta: map[string]interface{}{
-			"AgentName":    agent.GetCentralConfig().GetAgentName(),
-			"AgentVersion": cmd.BuildVersion,
+			"AgentName":       agent.GetCentralConfig().GetAgentName(),
+			"AgentVersion":    cmd.BuildVersion,
+			"AgentType":       agent.GetCentralConfig().GetAgentType().ToString(),
+			"AgentSDKVersion": config.SDKVersion,
```

Comment:
are these new ones not both available from the cmd.* args?

Reply:
I think the AgentType being used here is Traceability vs Discovery, what they want to see the BuildAgentName and the SDKBuildVersion from the cmd package.